source/copilot.txt

```diff
@@ -32,10 +32,17 @@ Commands
 The ``/query`` command assists in generating queries from a natural
```

Comment:
A bit above this, in the paragraph before, we have written:
```
Through GitHub Copilot's chat feature, users with |vsce| can interact
with their MongoDB clusters and generate code with MongoDB
domain-specific knowledge on top of GitHub Copilot's LLM.
```
Should we add something about the MongoDB documentation chatbot there as well?

Reply:
For the sake of simplicity, I would recommend only mentioning the documentation chatbot on the /docs page

Reply:
I did add a mention of learning about MongoDB in this section (referring to the /docs command)